example.env

```diff
@@ -1,2 +1,2 @@
 REACT_APP_API_URL=https://fallingfruit.org/api/0.2
-REACT_APP_API_KEY=
\ No newline at end of file
+REACT_APP_API_KEY=AKDJGHSD
```

Comment:
Can you remove the value from here? `example.env` just stores the keys so that anyone setting up the repo can fill them in in the actual `.env`.